front-end/src/pages/student/course/question/components/StudentMessage.module.css

```
@@ -0,0 +1,132 @@
+  }
+
+  .time {
+    font: var(--web-caption4-medium);
+  }
+}
+
+/*웹 뷰 */
+@media all and (min-width: 961px) {
+  .messageContainer {
+    min-width: 676x;
+    min-height: 134px;
```

Biome (1.9.4)

[error] Unexpected unknown unit: x

See MDN web docs for more details.
Use a known unit instead, such as:


(lint/correctness/noUnknownUnit)